bin/cli.js

```diff
@@ -40,7 +40,18 @@ function bump(type) {
     let newVersion;
 
     if (fs.existsSync('pom.xml')) {
+        const currentRev = execSync('hg id -i').toString().trim();
+        const currentBranch = execSync('hg id --branch').toString().trim();
+        const latestTag = execSync(`hg log -r "." -b ${currentBranch} --template "{latesttag}"`).toString().trim();
```

Comment:
Can you create 3 functions for those const instead of duplicating the code?

```
const getCurrentRev = () => execSync('hg id -i').toString().trim();
const getCurrentBranch = () => execSync('hg id --branch').toString().trim();
const getLatestTag = () => execSync(`hg log -r "." -b ${currentBranch} --template "{latesttag}"`).toString().trim();
```

Reply:
sure